meetup/services/meetup_api_response_parser.py

```diff
@@ -3,9 +3,27 @@ def parse(api_response):
   parsed_response = {}
 
   if json:
-    parsed_response['venue'] = json[0].get('venue').get('name')
-    parsed_response['next_event_topic'] = json[0].get('name')
-    parsed_response['datetime'] = json[0].get('time')
-    parsed_response['lat'] = json[0].get('group').get('lat')
-    parsed_response['lon'] = json[0].get('group').get('lon')
+
+    try:
+      parsed_response['venue'] = json[0].get('venue').get('name')
```

Comment:
...but a better refactoring is called the Null Object pattern: http://refactoring.com/catalog/introduceNullObject.html